### PR TITLE
DAH-1170 update translation and stylings for lottery modal

### PR DESCRIPTION
**Pull request by kramduckner**
[DAH-1170](https://sfgovdt.jira.com/jira/software/c/projects/DAH/boards/122?modal=detail&selectedIssue=DAH-1170)

- bump version of ui-components to include ExpandableContent className prop
- update translation string to mirror Angular implementation
- add padding around ExpandableContent button to mirror Angular implementation
- refactor ListingDetailsLotteryRanking to allow for stylings to mirror Angular Implementation
- refactor ListingDetailsLotteryRanking to use same implementation of lottery bucket details

This ticket addresses an [erroneous translation within the Lottery Rankings Modal](https://user-images.githubusercontent.com/5691923/173703877-ec161a6c-73a2-4210-bcd7-274cfd43e131.png) and making the styles of the Lottery Modal to be more in-line with Angular. While improving the styling inside of the ListingDetailsLotteryPreferences, it was noticed the lottery bucket details were implemented differently despite them being the same container, so those were consolidated and that fixed an additional styling discrepancy.

**Review Instructions**
- Ensure that the Expanded Button text and styling is the same in the react version as the angular version image
- Check the borders go to the full width of the lottery results modal beneath the "GENERAL POOL" section and "LIVE OR WORK IN SAN FRANCISCO PREFERENCE"
image
- Ensure the General Pool section padding/styling is consistent with other preference sections (e.g. CERTIFICATE OF PREFERENCE (COP)

### CODE REVIEW

**Comment by christine-sfg** [comment]
Code changes look great. Appreciate all the styling/polishing.

I'll do some manual/visual testing once the branch is up to date and review app is up.

**Comment by christine-sfg** [comment]
Was able to see the changes on the review app now. Looks great! :shipit:

File: package.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
     "@babel/plugin-transform-react-jsx": "^7.12.16",
     "@babel/preset-react": "^7.12.13",
     "@babel/preset-typescript": "^7.12.16",
-    "@bloom-housing/ui-components": "^4.4.1-alpha.25",
+    "@bloom-housing/ui-components": "4.4.1-alpha.26",
```
Comment (emilyjablonski) [inline package.json:11]: ❤️

**Review by christine-sfg — APPROVED** [review]

**Comment by christine-sfg** [comment]
lgtm. great job!

**Comment by christine-sfg** [comment]
Please move the review instructions to jira since design and pm will need this for product acceptance testing

**Pull request merged**